apps/publisher/config/publisher.json

```diff
@@ -66,6 +66,10 @@
             }
         }
     },
+    "metrics":{
+        "enabled":true,
+        "enableTraceLogs":true
```

Comment:
Thanks @lahirus , This has been already fixed in [this commit](https://github.com/wso2/carbon-store/commit/337853ffe82949a80091281f2dddba9f73d302c3)